README.md

```diff
@@ -1,5 +1,27 @@
 # digests
 
+The `digests` service serves content displayed at https://elifesciences.org/digests
+
+While an article [may contain a digest](https://elifesciences.org/articles/59007#digest), a digest is also 
+[a separate thing](https://elifesciences.org/digests/59007/a-long-history-of-lichen-mimicry) with a friendlier title, 
+a cover image and it's own production workflow managed by the Features team and coordinated using the elife-bot.
+
+The workflow to get content in to the `digests` service is roughly:
+
+1. Features team creates a zip file containing a digest .docx file and a digest image, and they place the zip file into a bucket.
+2. elife-bot runs an [IngestDigest workflow](https://github.com/elifesciences/elife-bot/blob/develop/workflow/workflow_IngestDigest.py) that:
+    * produces a simplifed `.docx` file and emails it to the Features team for production
+    * copies the digest image to an output bucket
+    * copies the `.docx` to an output bucket
+    * transforms the `.docx` to JATS XML and that is posted to the Typesetter's API endpoint
+3. a VoR article zip file is eventually ingested that contains the final digest and the elife-bot will include the `IngestDigestToEndpoint` activity that:
```

Comment:
3. and 4. are a great summary, those encapsulate the interactions with this project 👍